samples/gui/wasm-apps/increase/Makefile

```diff
@@ -25,9 +25,10 @@ SRCS += $(APP_FRAMEWORK_DIR)/wgl/app/src/*.c
 
 all:
 	@$(CC) $(CFLAGS) $(SRCS) \
-    --target=wasm32 -O3 -z stack-size=2048 -Wl,--initial-memory=65536 \
-    -Wl,--allow-undefined \
-    -Wl,--strip-all,--no-entry -nostdlib \
+    --target=wasm32-wasi -O3 -z stack-size=2048 -Wl,--initial-memory=65536 \
```

Comment:
Seems no need to add `--target=wasm32-wasi` now?

Reply:
Sorry, It is my fault. Inadvertently changed the Makefile.